apps/acceptance/clock/.jshintrc

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
A more correct file path would be: `apps/clock/test/acceptance` or `tests/acceptance/clock`. "acceptance" is not an app, which makes the file path odd.